oci/go.mod

```
@@ -0,0 +1,90 @@
+module github.com/defenseunicorns/pkg/oci
+
+go 1.22.1
```

```suggestion
go 1.21.8
```

ensure parity w/ `helpers/go.mod`.

For now this will have to be something we check manually (or write a small `bash` script) to ensure all the modules are the same Go version.